src/index.js

```diff
@@ -33,11 +33,13 @@ const searchFromQuerystring = (solrClient, options = {}) => {
     }
 
     // If the searchField is one for which we preserve state through qs.
+    // @TODO handle discrepancy between config and qs params for initial state
+    // IE site name values set in config but not present in querystring
     if (helpers.filterFieldsWithQsState.find((filterField) => filterField === searchField.field )) {
       searchField = helpers.qs.setFieldStateFromQs({
```

Comment:
This method sets query fields based on the qs param, since that is the mechanism that triggers searches (based on UI interactions).  We need to reconcile the facet value being set in config but not present in the QS param in order for this to work.  The solr client _should_ respond accordingly.

Reply:
It's not entirely clear to me _where_ we would do that @jesconstantine. I fiddled with this for an hour and kept getting notices like `params is read-only` when I tried to modify it conditionally.

Reply:
Found it!